src/app/_components/MarkdownContent.tsx

```diff
@@ -1,13 +1,59 @@
 import Image from 'next/image'
 
-import ReactMarkdown from 'react-markdown'
+import * as Sentry from '@sentry/node'
+import ReactMarkdown, { type Components } from 'react-markdown'
 import rehypeRaw from 'rehype-raw'
 import remarkGfm from 'remark-gfm'
 
+import { graphicsData } from '@/data/graphicsData'
+
 import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
 
+import { TextLink } from '@/components/TextLink'
+
 type MarkdownContentProps = {
-  children: string
+  children: Parameters<typeof ReactMarkdown>[0]['children']
+}
+
+const MarkdownImage: Components['img'] = ({ src, alt }) => {
+  const commonProps = {
+    quality: 100,
+    width: 800,
+    height: 450,
+    sizes: buildImageSizeProp({ startSize: '100vw', md: '660px' }),
```

Comment:
The `md` breakpoint could also use `MAX_WIDTH_OF_THE_CONTAINER`, if we end up creating it.